lib/hexpm_web/templates/package_report/_moderator_pannel.html.eex

```diff
@@ -0,0 +1,9 @@
+<div style="background-color: coral; width: 300px">
+    <h2>Moderator pannel </h2>
+    <%= form_tag Routes.package_report_path(Endpoint, :mod_review, @report_id) do %>
+        <input type="hidden" name="report_id" value="<%= @report_id %>"/>
+        <textarea name="comment"> </textarea>
+        <input type="submit" name="operation" value="Accept"/>
```

Comment:
I've created two separate controller actions as you suggested. They both perform the necessary updates through the same private function to prevent code duplication. I'm not sure if that fits your suggestion, specially in the "separate function in the context module" requirement.